### PR TITLE
ch-readme: Create ReadMe file and work QA

### PR DESCRIPTION
**Pull request by kunda4**
# PR Description

This Pull Request is About ReadMe file and working on Code review QA

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] Create  ReadMe file
- [x] Remove Unwanted dependencies
- [x] Working on code review QA

## Description of tasks that were expected to be completed and that are not yet completed

- [x] none

## How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependencies
2. run `npm run build`
3. run `npm start`
4. step 3
5. step 4...

## Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case it's appropriate and applicable

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: src/components/data.js
Comment (kareraolivier) [inline src/components/data.js:1]: Also this file will create conflict with the one who is working on them

**Comment by kareraolivier** [comment]
you can see that you do not have a preview link that netlify failed to deploy you changes!
you need to fix this as soon as possible

File: src/sections/skillSet.html
Comment (kareraolivier) [inline src/sections/skillSet.html:1]: After removing the body your project was down find a way to keep it up and plz make sure the file you did not touch stay the same

File: .gitignore [hunk 1]
```diff
@@ -16,11 +16,11 @@ pids
 *.pid
 *.seed
 *.pid.lock
-dist/
```
Comment (kareraolivier) [inline .gitignore]: you did not add package.lock.json or vscode folder that contain settings and you removed all dist folder
look on this file again

File: index.html [hunk 2]
```diff
@@ -1,66 +1,70 @@
...
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>team project</title>
-    <link rel="stylesheet" type="text/css" href="./dist/output.css">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
```
Comment (kareraolivier) [inline index.html]: You were not supposed to touch this file

**Review by kareraolivier — APPROVED** [review]
ok

**Review by devfabien — APPROVED** [review]
Looking good

**Pull request merged**